### PR TITLE
Feature/smar 86

### PR DESCRIPTION
**Pull request by RaznoOleg**
## Description changes:
• Made frontend for patient card.

### General
- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Don't have "any" on my code
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside ternary operator
- [x] Don't have commented code
- [x] All links in on ENV files
- [x] Used camelCase for variables and functions
- [x] Functions is public only if it's used outside the class
- [x] No hardcoded values
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend
- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

#### View of patient information:
image

#### Show more/less button:
image
image

### CODE REVIEW

File: src/components/Patient/styles.ts [hunk 1]
```diff
@@ -41,3 +36,80 @@ export const LinkContainer = styled.div`
...
+  align-items: center;
+  margin-bottom: 8px;
+  font-size: ${NORMAL_FONT_SIZE};
+`;
+
+export const UserInfo = styled.p`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #060d33;
```
Comment (lyudad) [inline src/components/Patient/styles.ts]: ?

File: src/components/Patient/styles.ts [hunk 2]
```diff
@@ -41,3 +36,80 @@ export const LinkContainer = styled.div`
...
+  margin-top: 20px;
+  color: #060d33;
+  font-size: ${NORMAL_FONT_SIZE};
+`;
+
+export const LastAppointmentTitle = styled.span`
+  margin-right: 4px;
+  font-weight: 700;
+  line-height: 1.43;
+  color: #081972;
```
Comment (lyudad) [inline src/components/Patient/styles.ts]: ?

File: src/components/Patient/styles.ts [hunk 3]
```diff
@@ -41,3 +36,80 @@ export const LinkContainer = styled.div`
...
+`;
+
+export const LastAppointment = styled.p`
+  margin-bottom: 6px;
+  font-size: ${SMALL_FONT_SIZE};
+  line-height: 1.4;
+  color: #060d33;
```
Comment (lyudad) [inline src/components/Patient/styles.ts]: ?

File: src/components/Patient/styles.ts (hunk 1 above)
Comment (RaznoOleg) [reply src/components/Patient/styles.ts]: fixed

File: src/components/Patient/styles.ts (hunk 2 above)
Comment (RaznoOleg) [reply src/components/Patient/styles.ts]: fixed

File: src/components/Patient/styles.ts (hunk 3 above)
Comment (RaznoOleg) [reply src/components/Patient/styles.ts]: fixed

**Review by lyudad — APPROVED** [review]

**Pull request merged**